repo/js/StarfieldRaimentCollection/main.js

```
@@ -267,8 +320,35 @@
                     await genshin.returnMainUi();
                     break;
                 case "奇域：配队界面":
+                    let c1Ro = RecognitionObject.TemplateMatch(file.ReadImageMatSync("assets/c1.png"), 134, 93, 46, 51);
```

Biome (2.5.5)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)